Makefile

```diff
@@ -187,6 +189,10 @@ dependencies:
 ## Run cmd/addon-operator-manager against $KUBECONFIG.
 run-addon-operator-manager:
 
+## Generates the OLM bundle 
+generate-bundle:
+	$(PROJECT_DIR)/$(DEPENDENCIES)/bin/operator-sdk  generate bundle --input-dir $(PROJECT_DIR)/deploy --version 1.0.0
```

Comment:
Not having `--overwrite` here will skip bundle creation, I understand the `createdAt` field is a pinpoint but a fresh bundle generation is mandatory on every build.